app-frontend/app/src/links.ts

```diff
@@ -0,0 +1,11 @@
+import {Category} from "./model";
+
+const data: Category[] = require("../links.json");
```

Comment:
Could the type from `model.tsx` be reused here? (`type Links = Category[]`).

Reply:
Yes